integration/js/ContentShareOnlyAllowTwoTest.js

```diff
@@ -15,7 +20,16 @@ const { v4: uuidv4 } = require('uuid');
  */
 class ContentShareOnlyAllowTwoTest extends SdkBaseTest {
   constructor(name, kiteConfig) {
-    super(name, kiteConfig, "ContentShareOnlyAllowTwoTest");
+    super(name, kiteConfig, 'ContentShareOnlyAllowTwoTest');
+  }
+
+  // Commented out the UserJoinedMeetingCheck because we already check it in the JoinMeetingStep
+  async addUserToMeeting(attendee_id, sessionInfo) {
```

Comment:
Checking if this method is added, because SDKTestUtils one has that last `UserJoinedMeetingCheck`?

Reply:
that utils method is used in a lot of other places but I needed to change it, so instead of changing that, I created a new one here without one step